utilities/rpp-unittests/HOST_NEW/testAllScript_now.sh

```diff
@@ -0,0 +1,433 @@
+    echo "--------------------------------"
+    for ((bitDepth=0;bitDepth<7;bitDepth++))
+    do
+        printf "\n\n\nRunning New Bit Depth...\n-------------------------\n\n"
+        for ((outputFormatToggle=0;outputFormatToggle<2;outputFormatToggle++))
+        do
+
+            if [[ "$case" -eq 74 ]]
+            then
+                SRC_FOLDER_1_TEMP="$DEFAULT_HARRIS_CORNER_DETECTOR_IMAGES"
+                SRC_FOLDER_2_TEMP="$DEFAULT_HARRIS_CORNER_DETECTOR_IMAGES"
+            elif [[ "$case" -eq 75 ]]
+            then
+                SRC_FOLDER_1_TEMP="$DEFAULT_FAST_CORNER_DETECTOR_IMAGES"
+                SRC_FOLDER_2_TEMP="$DEFAULT_FAST_CORNER_DETECTOR_IMAGES"
+            elif [[ "$case" -eq 77 ]]
+            then
+                SRC_FOLDER_1_TEMP="$DEFAULT_HOUGH_LINES_IMAGES"
+                SRC_FOLDER_2_TEMP="$DEFAULT_HOUGH_LINES_IMAGES"
+            elif [[ "$case" -eq 78 ]]
+            then
+                SRC_FOLDER_1_TEMP="$DEFAULT_HOG_IMAGES"
+                SRC_FOLDER_2_TEMP="$DEFAULT_HOG_IMAGES"
+            else
+                SRC_FOLDER_1_TEMP="$SRC_FOLDER_1"
+                SRC_FOLDER_2_TEMP="$SRC_FOLDER_2"
+            fi
+
+            printf "\n./BatchPD_host_pkd3 $SRC_FOLDER_1_TEMP $SRC_FOLDER_2_TEMP $DST_FOLDER_TEMP $bitDepth $outputFormatToggle $case 0"
+            ./BatchPD_host_pkd3 "$SRC_FOLDER_1_TEMP" "$SRC_FOLDER_2_TEMP" "$DST_FOLDER_TEMP" "$bitDepth" "$outputFormatToggle" "$case" "0" | tee -a "$LOGGING_FOLDER/BatchPD_host_pkd3_host_raw_performance_log.txt"
+
+            if [ "$case" -eq 8 ]
+            then
+                for ((noiseType=0;noiseType<3;noiseType++))
+                do
+                    printf "\n./Tensor_host_pkd3 $SRC_FOLDER_1_TEMP $SRC_FOLDER_2_TEMP $DST_FOLDER_TEMP $bitDepth $outputFormatToggle $case $noiseType 0"
+                    ./Tensor_host_pkd3 "$SRC_FOLDER_1_TEMP" "$SRC_FOLDER_2_TEMP" "$DST_FOLDER_TEMP" "$bitDepth" "$outputFormatToggle" "$case" "$noiseType" "0"
+                done
+            elif [ "$case" -eq 21 ]
+            then
+                for ((interpolationType=0;interpolationType<6;interpolationType++))
+                do
+                    printf "\n./Tensor_host_pkd3 $SRC_FOLDER_1_TEMP $SRC_FOLDER_2_TEMP $DST_FOLDER_TEMP $bitDepth $outputFormatToggle $case $interpolationType 0"
+                    ./Tensor_host_pkd3 "$SRC_FOLDER_1_TEMP" "$SRC_FOLDER_2_TEMP" "$DST_FOLDER_TEMP" "$bitDepth" "$outputFormatToggle" "$case" "$interpolationType" "0"
+                done
+            else
+                printf "\n./Tensor_host_pkd3 $SRC_FOLDER_1_TEMP $SRC_FOLDER_2_TEMP $DST_FOLDER_TEMP $bitDepth $outputFormatToggle $case ${num_iterations} ${test_type} ${pkd_layout} 0"
+                ./Tensor_host_pkd3 "$SRC_FOLDER_1_TEMP" "$SRC_FOLDER_2_TEMP" "$DST_FOLDER_TEMP" "$bitDepth" "$outputFormatToggle" "$case" "$num_iterations" "$test_type" "$pkd_layout" "0" | tee -a "$LOGGING_FOLDER/Tensor_host_pkd3_host_raw_performance_log.txt"
```

Comment:
Generate the name for log file (Ex: Tensor_host_pkd3_host_raw_performance_log.txt) based on the layout value based on the layout value